src/SFA.DAS.LevyTransferMatching.Domain/Extensions/PledgeStatusExtenstions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SFA.DAS.LevyTransferMatching.Domain.Types;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.LevyTransferMatching.Domain.Extensions
+{
+    public static class PledgeStatusExtenstions
+    {
+        public static string GetLabelForPledgeStatus(this PledgeStatus status)
+        {
+            switch (status)
+            {
+                case PledgeStatus.Active: return "ACTIVE";
+                case PledgeStatus.Closed: return "CLOSED";
+                default:
+                    return string.Empty;
+            }
```

Comment:
Not mandatory but if you want, above switch can be simplified to:
            return status switch
            {
                PledgeStatus.Active => "ACTIVE",
                PledgeStatus.Closed => "CLOSED",
                _ => string.Empty,
            };

Reply:
@SHESFA Thanks, your suggestion looks really nice but I just kept it like in the other extension methods. For example: GetLabelForSender(this ApplicationStatus status) has the same structure. 

I also tested the code at the time of PR and it was all working fine.